lib/sidekiq/processor.rb

```diff
@@ -119,18 +119,20 @@ def dispatch(job_hash, queue)
 
       @job_logger.with_job_hash_context(job_hash) do
         @retrier.global(pristine, queue) do
-          @job_logger.call(job_hash, queue) do
-            stats(pristine, queue) do
-              # Rails 5 requires a Reloader to wrap code execution.  In order to
-              # constantize the worker and instantiate an instance, we have to call
-              # the Reloader.  It handles code loading, db connection management, etc.
-              # Effectively this block denotes a "unit of work" to Rails.
-              @reloader.call do
-                klass = constantize(job_hash["class"])
-                worker = klass.new
-                worker.jid = job_hash["jid"]
-                @retrier.local(worker, pristine, queue) do
-                  yield worker
+          @job_logger.with_log_level(job_hash["log_level"]) do
```

Comment:
Can you combine the log_level and hash_context setup into one method since the job_hash is already being passed in?